src/main/java/com/o19s/es/ltr/ranker/ranklib/RankLibScriptEngine.java

```diff
@@ -46,7 +46,7 @@
     private final LtrRankerParserFactory factory;
 
     public RankLibScriptEngine(Settings settings, LtrRankerParserFactory factory) {
-        super(settings);
```

Comment:
if settings is no longer accepted in the parent constructor we should remove it from RankLibScriptEngine constructor as well.